corehq/apps/integration/payments/schemas.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from typing import Literal
+
+
+@dataclass
+class PartyDetails:
```

Comment:
nit: Is party the term used by MoMo? I see you have used payee for it in `PaymentTransferDetails` which seems nicer.

Reply:
Yes, "party" is MTN MoMo slang.